github/table_github_team_repository.go

```diff
@@ -0,0 +1,101 @@
+package github
+
+import (
+	"context"
+
+	"github.com/google/go-github/v33/github"
+
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func gitHubTeamRepositoryColumns() []*plugin.Column {
+	repoColumns := gitHubRepositoryColumns()
+	teamColumns := []*plugin.Column{
+		{Name: "organization", Type: pb.ColumnType_STRING, Description: "The organization the team is associated with.", Transform: transform.FromQual("organization")},
+		{Name: "slug", Type: pb.ColumnType_STRING, Description: "The team slug name.", Transform: transform.FromQual("slug")},
+	}
+
+	return append(repoColumns, teamColumns...)
+}
+
+func tableGitHubTeamRepository() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_team_repository",
+		Description: "GitHub Repositories that a given team is associated with. GitHub Repositories contain all of your project's files and each file's revision history.",
+		List: &plugin.ListConfig{
```

Comment:
Is it possible to add a Get function here, which takes in an organization, slug, and repository full name? You may be able to use this API - https://docs.github.com/en/rest/teams/teams#check-team-permissions-for-a-repository, though in its sample response, it looks like it has more fields than its list counterpart (https://docs.github.com/en/rest/teams/teams#list-team-repositories).